x-pack/plugin/voting-only-node/src/main/java/org/elasticsearch/cluster/coordination/votingonly/VotingOnlyNodeFeatureSet.java

```diff
@@ -25,22 +24,7 @@
 import org.elasticsearch.xpack.core.action.XPackUsageFeatureTransportAction;
 import org.elasticsearch.xpack.core.votingonly.VotingOnlyNodeFeatureSetUsage;
 
-public class VotingOnlyNodeFeatureSet implements XPackFeatureSet {
-
-    @Override
-    public String name() {
-        return XPackField.VOTING_ONLY;
-    }
-
-    @Override
-    public boolean available() {
-        return true;
-    }
-
-    @Override
-    public boolean enabled() {
-        return true;
-    }
+public class VotingOnlyNodeFeatureSet {
```

Comment:
Nit: move the inner TransportAction classes to VotingOnlyXxxTransportAction top-level classes and get rid of this? The name is confusing (still refers to FeatureSet)